<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPIC32UBLDlg</class>
 <widget class="QMainWindow" name="CPIC32UBLDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PIC32UBL</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid gray;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Communication Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="title">
               <string>Serial Port</string>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>20</y>
                 <width>71</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>Com Port</string>
               </property>
              </widget>
              <widget class="QComboBox" name="ctrlComboBoxComPort">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>111</width>
                 <height>25</height>
                </rect>
               </property>
              </widget>
              <widget class="QComboBox" name="ctrlComboBoxBaudRate">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>40</y>
                 <width>121</width>
                 <height>25</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>140</x>
                 <y>20</y>
                 <width>81</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>Baud Rate</string>
               </property>
              </widget>
              <widget class="QCheckBox" name="ctrlCheckBoxComEnable">
               <property name="geometry">
                <rect>
                 <x>260</x>
                 <y>40</y>
                 <width>90</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Enable</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="title">
               <string>USB</string>
              </property>
              <widget class="QComboBox" name="ctrlComboBoxVIDPID">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>241</width>
                 <height>25</height>
                </rect>
               </property>
              </widget>
              <widget class="QCheckBox" name="ctrlCheckBoxUSBEnable">
               <property name="geometry">
                <rect>
                 <x>260</x>
                 <y>40</y>
                 <width>90</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Enable</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>20</y>
                 <width>71</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>VID:PID</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Ethernet</string>
              </property>
              <widget class="QCheckBox" name="ctrlCheckBoxEthEnable">
               <property name="geometry">
                <rect>
                 <x>260</x>
                 <y>40</y>
                 <width>90</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Enable</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_4">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>20</y>
                 <width>71</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>IP Address</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="ctrlEditBoxIPaddress">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>241</width>
                 <height>25</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_5">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>70</y>
                 <width>71</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>UDP Port</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="ctrlEditBoxSocketaddress">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>90</y>
                 <width>111</width>
                 <height>25</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QPushButton" name="ctrlButtonBootloaderVer">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>121</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Bootloader Ver.</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ctrlButtonLoadHex">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>121</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Load Hex File</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ctrlButtonErase">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>10</y>
           <width>121</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Erase</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ctrlButtonProgram">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>121</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Program</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ctrlButtonVerify">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>121</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Verify</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ctrlButtonRunApplication">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>121</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Run Application</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ctrlButtonEraseProgVerify">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>251</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Erase-Program-Verify</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ctrlButtonConnectDevice">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>90</y>
           <width>121</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="ctrlProgressBar">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>130</y>
           <width>381</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="ctrlEditConsole">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>170</y>
           <width>381</width>
           <height>181</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
